### PR TITLE
Readme update node 18 16 1

### PR DESCRIPTION
**Pull request by Duncan-Brain**
**Description**

This update takes some node related suggestions from https://github.com/LiteFarmOrg/LiteFarm/pull/2484 and incorporates them with this readme in three steps:

- adding .nvmrc to each folder containing packages (excluding new ci/test folders)
- update readme to remove specific node version references (minus troubleshooting)
- add some extra info to .env.default to cover some parts of PR2484

Jira link:

**Type of change**

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

**How Has This Been Tested?**

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Passes test case
- [ ] UI components visually reviewed on desktop view
- [ ] UI components visually reviewed on mobile view
- [ ] Other (please explain)

**Checklist:**

- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] The precommit and linting ran successfully
- [ ] I have added or updated language tags for text that's part of the UI
- [ ] I have added "MISSING" for all new language tags to languages I don't speak
- [ ] I have added [the GNU General Public License](https://lite-farm.atlassian.net/l/cp/BT0Dd7WW) to all new files

### CODE REVIEW

**Review by kathyavini — COMMENTED** [review]
Thanks for updating this! I like the `nvm use` strategy and I think it's much improved overall. Comments I left were very minor.

File: README.md [hunk 1]
```diff
@@ -285,6 +282,23 @@ Use cases in which we currently utilize docker at LiteFarm include:
...
+   npm ERR! ERESOLVE could not resolve
+   npm ERR!
+   npm ERR! While resolving: objection@2.2.17...
+   ```
+
+   Previously this package required the use of node 16.15.0 so `nvm use 16.15.0` would fix the issue. But this should no longer be visible . Use nvm to install and use the Node version 16.15.0 with the commands, `nvm install 16.15.0` then `nvm use 16.15.0`. Then try again.
+
+2. On Windows: During `npm run nodemon` of `/packages/api` there is an error - possibly with the client variable of knex.
+
+   The NODE_ENV variable is not being set properly you will need to adjust the nodemon script in `/packages/api/package.json`
```
[the remark this answers is not in the document]
Comment (Duncan-Brain) [reply README.md:300]: We definitely can, I have a related Jira ticket - but could make a new one : https://lite-farm.atlassian.net/jira/software/c/projects/LF/issues/LF-2844 

The ticket already made got complicated when certain elements of CI used it -for example cypress automated tests didn't run.

File: README.md [hunk 2]
```diff
@@ -285,6 +282,23 @@ Use cases in which we currently utilize docker at LiteFarm include:
 - create a `.env` file at the root directory of the project i.e. LiteFarm
 - Add key-value pairs in the `.env` by referring to the `docker-compose.[ENV].yml` that contains the docker env keys.
 
+## Troubleshooting
+
+1. DEPRECATED: During `npm install` of `/packages/api` you see the following:
```
[the remark this answers is not in the document]
Comment (Duncan-Brain) [reply README.md:287]: Makes sense to me!

File: README.md [hunk 3]
```diff
@@ -22,22 +22,27 @@ LiteFarm is comprised of three applications which all reside in this monorepo.
...
-2. Check to see if you have pnpm installed. On a Mac use the command `pnpm -v`. If it is installed, the version will be reported. If you do not have it installed, run `npm install -g pnpm` in a terminal.
-3. Check to see if you have NVM installed. On a Mac use the command `nvm -v`. If you do not have NVM (Node Version Manager) installed, install it using these instructions: [NVM](https://www.loginradius.com/blog/engineering/run-multiple-nodejs-version-on-the-same-machine/)
+1. Check to see if you have Node.js installed. We use the version specified in the `.nvmrc` file of each folder with package imports. On a Mac use the command `node-v` in terminal. If it is installed, the version in use will be reported in the terminal. If not, install it from [node.js](https://nodejs.org/en/download/package-manager/).
+2. Check to see if you have NVM installed. On a Mac use the command `nvm -v`. If you do not have NVM (Node Version Manager) installed, install it using these instructions: [NVM](https://www.loginradius.com/blog/engineering/run-multiple-nodejs-version-on-the-same-machine/)
```
[the remark this answers is not in the document]
Comment (Duncan-Brain) [reply README.md:26]: I think thats a good question - I started removing that line but then decided not to rock the boat right now. 

I wondered if having at least one version was necessary for nvm. Or if nvm works the same for everyone. Or for beginner developers who are installing for the first time if it's helpful or not to mention we are using node.js. 

I am down but yeah decided not to iron out the details immediately.

File: README.md [hunk 4]
```diff
@@ -237,7 +234,7 @@ You can also test LiteFarm on your actual mobile device using the network adddre
 
 Please see https://ngrok.com/ for more general information about ngrok.
 
-Use cases in which we currently utilize ngrok at LiteFarm include:
+While not required for most developers. Use cases in which we currently utilize ngrok at LiteFarm include:
```
[the remark this answers is not in the document]
Comment (Duncan-Brain) [reply README.md:237]: Will fix but .. https://www.youtube.com/watch?v=P_i1xk07o4g haha

**Review by kathyavini — APPROVED** [review]

**Review by kathyavini — APPROVED** [review]

**Review by SayakaOno — APPROVED** [review]
Looks great!

**Pull request merged**